src/modules/createGame.js

```diff
@@ -0,0 +1,12 @@
+async function create() {
```

Comment:
- Kindly Use arrow functions instead of the function keyword as stated in the [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md) to make your code cleaner 👍

Reply:
This change has been done well